OpenSim/Moco/MocoGoal/MocoContactImpulseTrackingGoal.cpp

```diff
@@ -117,23 +111,25 @@ void MocoContactTrackingGoal::initializeOnModelImpl(const Model& model) const {
     const ExternalLoads* extLoads;
     if (!getProperty_external_loads().empty()) {
         OPENSIM_THROW_IF_FRMOBJ(!getProperty_external_loads_file().empty(),
-                Exception,
-                "Expected either an ExternalLoads file or object, but both "
-                "were provided.");
+            Exception,
+            "Expected either an ExternalLoads file or object, but both "
+            "were provided.");
         extLoads = &get_external_loads();
-    } else if (!get_external_loads_file().empty()) {
+    }
+    else if (!get_external_loads_file().empty()) {
         extLoadsFromFile = OpenSim::make_unique<ExternalLoads>(
-                get_external_loads_file(), true);
+            get_external_loads_file(), true);
         extLoads = extLoadsFromFile.get();
-    } else {
+    }
+    else {
         OPENSIM_THROW_FRMOBJ(Exception, "No ExternalLoads provided.");
     }
 
     // Spline the data.
     const std::string dataFilePath =
-            convertRelativeFilePathToAbsoluteFromXMLDocument(
```

Comment:
Looks like some auto-formatting happened, which is fine, but make sure that no whitespace tabs have been entered into the code. If so, replace them with spaces. (Visual Studio has an option to do this for you).